### DISCUSSION TITLE
App did not run well

### DISCUSSION THREAD

**Discussion by successbyte** [quran/quran.com-frontend-next#2249] [Q&A]
Hi,
I am interested in the contributing in quran.com
I did all the Running the App Locally steps in readme file and worked well then changed the .env.example  to .env.local and then run the app it runs but when clock on a surah (chapter) it shows the bellow page 

![image](https://github.com/user-attachments/assets/bc14a44f-3d86-4e64-b65a-5b1ada1ae703)

could anyone please help me how to run the app?

**Comment by arishgithub** [comment]
I think the maintainers of open source of this repo dont look at these at all, this question is almost 1.5 years old and has no answers yet, I have setup the repo today and facing the same issue.
